pkg/setting/setting_smtp.go

```diff
@@ -14,13 +19,17 @@ type SmtpSettings struct {
 	EhloIdentity   string
 	StartTLSPolicy string
 	SkipVerify     bool
+	StaticHeaders  map[string]string
 
 	SendWelcomeEmailOnSignUp bool
 	TemplatesPatterns        []string
 	ContentTypes             []string
 }
 
-func (cfg *Cfg) readSmtpSettings() {
+// validates smtp headers
+var smtpHeaderRegex = regexp.MustCompile(`^[A-Z][A-Za-z0-9]*(-[A-Z][A-Za-z0-9]*)*$`)
```

Comment:
_technically_ these are "mail headers" - they're not part of the SMTP protocol (they're also not strictly MIME headers, as they don't relate to MIME directly, and are not part of multipart messages)

```suggestion
// validates mail headers
var mailHeaderRegex = regexp.MustCompile(`^[A-Z][A-Za-z0-9]*(-[A-Z][A-Za-z0-9]*)*$`)
```

Reply:
I should also say, my (incredibly pedantic) suggestion is mainly intended for internal variable naming. I think it still makes sense to name the config section `smtp.static_headers`, for example, because it only relates to messages sent over SMTP 😅